src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -429,5 +429,89 @@ public static String getCopyrightAcceptUrl(String copyrightAlert,
 		}
 		return contentUrl;
 	}
+	
+	/**
+	 * change folder path based on updated folder title
+	 * @param folderNameUpdates
+	 * @param title
+	 * @param folderName
+	 * @return
+	 */
+	public static HashMap<String, String> updateFolderName(HashMap<String, String> folderNameUpdates,
+			String title, String folderName) {
+		// update folder name if there is any parent folder renaming
+		// checks for folder name updates in the path
+		// replace all old folder title with new title
+		String currentFolderName = folderName;
+		for (String oldFolderName : folderNameUpdates.keySet()) {
+			if (folderName.startsWith(oldFolderName))
```

Comment:
Does this deal with only the top level folder?  Can there be nested subfolders that matter?